### PR TITLE
EVPN-VxLAN Multihoming HLD

### PR DESCRIPTION
**Pull request by mikemallin**
This HLD introduces support for EVPN-VXLAN Multihoming on SONiC.

### CODE REVIEW

**Comment by Yuval-Mellanox** [comment]
@mikemallin is this proposal based on the previous community proposal for MH (https://github.com/sonic-net/SONiC/issues/1412) or is it a fresh new one?

**Comment by adyeung** [comment]
@Yuval-Mellanox @zhangyanzhao the EVPN MH proposals from Cisco and BRCM are under active review in the Routing WG, the eventual goal is to reach a converged solution for the extended community, I haven't seen a HLD from #1412 yet, but if any org or individual is interested in contributing to EVPN MH, please reachout to the Routing WG sonic-wg-routing@lists.sonicfoundation.dev

**Comment by mikemallin** [comment]
> @mikemallin is this proposal based on the previous community proposals for MH (#1412) Marvell (#1622) Broadcom or is it a fresh new one? @zhangyanzhao we should converge into one proposal

This is the proposal that has been presented by Cisco at the Routing WG a few weeks ago.

File: doc/vxlan/EVPN/EVPN_VXLAN_MULTIHOMING.md [hunk 1]
```diff
@@ -0,0 +1,1140 @@
...
+The following CLI configuration becomes the entries in CONFIG DB below:
+
+```
+config evpn_es add PortChannel01 00:11:22:33:44:55:66:77:88:99
+config evpn_es add PortChannel02 3 00:11:22:33:44:55
+```
+
+```
+"ETHERNET_SEGMENT_TABLE:PortChannel01": {
+  "interface"                 = "PortChannel01"
```
Comment (AntonButenkoGL) [inline doc/vxlan/EVPN/EVPN_VXLAN_MULTIHOMING.md:564]: Why do we need the same "PortChannel01" in the table name as "interface" field?

File: doc/vxlan/EVPN/EVPN_VXLAN_MULTIHOMING.md [hunk 2]
```diff
@@ -0,0 +1,1140 @@
...
+
+<a id="312-scalability-requirements"></a>
+### 3.1.2 Scalability Requirements
+
+There are no additional scalability requirements beyond those in section 2.1.2.
+
+<a id="313-sai-overview"></a>
+### 3.1.3 SAI Overview
+
+Add a new attribute "SAI_BRIDGE_PORT_ATTR_DF_BLOCK" in sai_bridge_port_attr_t to indicate that the bridge port is blocked to forward any BUM traffic received from the Tunnel.
```
Comment (AntonButenkoGL) [inline doc/vxlan/EVPN/EVPN_VXLAN_MULTIHOMING.md:631]: As soon as I understand - we support only VLAN-VNI mapping and not the bridging between vxlan interface and some physical port. 
If this is true - we don't need this requirement, right?

**Review by AntonButenkoGL — COMMENTED** [review]
Hello,

We have a small team of engineer working on the EVPN Multihoming implementation and decided to start from this PR as requirements base. 

Can you, please, look at the questions we have regarding the requirements?

**Comment by adyeung** [comment]
Converged HLD posted as #1702, closing this PR

**Pull request closed without merging**